ascii-dev/mt-react
# #160935562 Setup CodeClimate & Coveralls

#### What does this PR do?
To setup CodeClimate & Coveralls for the application.
#### Description of Task to be completed?
- setup codeclimate and coveralls config
#### How should this be manually tested?
After cloning the repo and running `npm install`
You should see the `.coveralls.yml` file in the project
#### What are the relevant pivotal tracker stories?
[#160935562](https://www.pivotaltracker.com/story/show/160935562)
#### Questions
N/A
